flash/vision/backbones.py

```diff
@@ -29,7 +32,11 @@
     import torchvision
 
 if _BOLTS_AVAILABLE:
-    from pl_bolts.models.self_supervised import SimCLR, SwAV
+    if os.getenv("WARN_MISSING_PACKAGE") == "0":
```

Comment:
what is this about? is it necessary for docs?

Reply:
Just resolved a warning issue stressed by Will.